contractsV5/bridges/bancor/KyberBancorReserve.sol

```diff
@@ -15,35 +15,31 @@ contract KyberBancorReserve is IKyberReserve, Withdrawable, Utils {
     bool public tradeEnabled;
     uint public feeBps;
 
-    IBancorNetwork public bancorNetwork; // 0x0e936B11c2e7b601055e58c7E32417187aF4de4a
+    IBancorNetwork public bancorNetwork; // 0x3ab6564d5c214bc416ee8421e05219960504eead
 
-    IERC20 public bancorEth; // 0xc0829421C1d260BD3cB3E0F06cfE2D52db2cE315
+    // IERC20 public bancorEth; // 0xc0829421C1d260BD3cB3E0F06cfE2D52db2cE315
```

Comment:
I keep the comments so when I deploy I could just copy the value

Reply:
sooner or later this data will be wrong
lets not have out dated comments in production code.
please see where it can be placed
maybe deploy script is more suitable.
it can be added to web3 deployment folder.

Reply:
Sure, I'll remove these comments